due-processed/page-about.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+get_header();
+?>
+
+	<main id="primary" class="site-main">
```

Comment:
You could add a class here for about page and target that instead of targeting the page id in the stylesheet
```suggestion
	<main id="primary" class="site-main site-main-about">
```